server/datastore/mysql/migrations/tables/20220921090715_CreateOperatingSystemVulnerabilitiesTable.go

```diff
@@ -0,0 +1,42 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220921090715, Down_20220921090715)
+}
+
+func Up_20220921090715(tx *sql.Tx) error {
+	logger.Info.Println("Creating table operating_system_vulnerabilities...")
+
+	_, err := tx.Exec(`
+		CREATE TABLE operating_system_vulnerabilities
+		(
+			id                  INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+			host_id             INT UNSIGNED NOT NULL,
+			operating_system_id INT UNSIGNED NOT NULL,
+			cve                 VARCHAR(255) NOT NULL,
+			source              SMALLINT              DEFAULT 0,
+			created_at          TIMESTAMP    NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+			UNIQUE KEY idx_operating_system_vulnerabilities_unq_cve (host_id, cve),
+			INDEX idx_operating_system_vulnerabilities_operating_system_id_cve (operating_system_id, cve),
+			INDEX idx_operating_system_vulnerabilities_cve (cve)
```

Comment:
I don't think this index is needed, right?

Reply:
Not right now, but I still don't know how we are going to present OS vulnerabilities on the UI - if we are going to add a new page for installed OS (same way we do for software) and we allow the user to search by CVE then this index will be needed. I figured is easier to remove an index that to add a new one and since this table won't be that big, I don't think this adds much overhead. 

But I guess that I should follow YAGNI here ... I'll remove it

Reply:
Removed